cluster/gce/util.sh

```diff
@@ -2132,11 +2132,13 @@ function kube-up() {
     write-cluster-name
     create-autoscaler-config
     create-master
-    create-nodes-firewall
-    create-nodes-template
-    # Windows nodes take longer to boot and setup so create them first.
-    create-windows-nodes
-    create-linux-nodes
+    if [[ "${KUBE_CREATE_NODES}" == "true" ]]; then
+      create-nodes-firewall
+      create-nodes-template
```

Comment:
i'm wondering if we shouldn't create firewall and template. Because that means that we can't scale this cluster up later.
Maybe that's fine, but we should probably document it clearly what the contract is.

Reply:
Good point. Moved this outside of if.